src/cli/components/landingUI.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Text } from "ink";
+import * as path from "path";
+import { isFilePresent } from "../utils/fileSystem";
+import { FileStatus, TestFile } from "../../types/exercises.types";
+
+const Landing: React.FC<TestFile> = ({ testFileName }) => {
+    if (testFileName) {
+        const defaultExercisePath = path.join(
+            process.cwd(),
+            "exercises",
+            testFileName.toLowerCase(),
+        );
+        const fileStatus: FileStatus = isFilePresent(
+            defaultExercisePath,
+            testFileName,
+        );
+        if (fileStatus === FileStatus.SUCCESS) {
+            return (
+                <Text>
+                    Hello <Text color="green">{testFileName}</Text>
+                </Text>
+            );
+        } else if (fileStatus === FileStatus.INCONCLUSIVE) {
+            return (
+                <Text>
+                    <Text color="red">{testFileName}</Text> is inconclusive,
```

Comment:
```suggestion
                    There exists more than one files with the name <Text color="red">{testFileName}</Text>
```
This might be clearer than "inconclusive"

Reply:
Nice catch!